cmd/kubectl-gadget/utils/trace.go

```diff
@@ -290,32 +455,158 @@ RetryLoop:
 		// Don't print warnings if at least one node succeeded. This avoids showing
 		// warnings together with the actual output generated by other nodes.
 		if successNodeCount == 0 {
-			printTraceFeedback(contextLogger.Warningf, nodeWarnings)
+			printWarningFunction := func(format string, args ...interface{}) { fmt.Printf(format, args) }
```

Comment:
I think warnings should also go to stderr, and it should be prefixed with WARNING?